### DISCUSSION TITLE
Documentos para inscrição via empenho - Python Cerrado

### DISCUSSION THREAD

**Discussion by anadulce** [apyb/comunidade#173] [Pergunta]
Boa tarde! Tudo bem?

Estou ajudando na organização da Python Cerrado, e foi solicitado o preenchimento da seguinte declaração, em papel timbrado da APyB, como parte da documentação para uso de inscrição via empenho. Poderiam preencher, por favorzinho?

https://docs.google.com/document/d/11TC9UhdfKw8NCfgb0WDb_u9070b8Ebgh/edit?usp=sharing&ouid=105012949324751328317&rtpof=true&sd=true

@apyb/diretoria

**Comment by anicelysantos** [comment]
@anadulce aqui está: https://drive.google.com/file/d/1bb5KNCDW07TBklFfELiqiRVEM3b7ElQO/view?usp=drive_link

**Reply by anadulce** [reply]
Muuuito obrigada, @anicelysantos <3
